src/cordova/apple/OutlineAppleLib/Sources/OutlineTunnelSources/SwiftSources/Subnet.swift

```diff
@@ -16,8 +16,8 @@ import Foundation
 
 // Represents an IP subnetwork.
 @objcMembers
-class Subnet: NSObject {
-  static let kReservedSubnets = [
+public class Subnet: NSObject {
```

Comment:
Perhaps add a comment explaining that we need to make the methods public so we can access from Objective C?